libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -330,7 +333,28 @@ def _stream(
             result = self._generate(
                 messages, stop=stop, run_manager=run_manager, **kwargs
             )
-            yield cast(ChatGenerationChunk, result.generations[0])
+            message = result.generations[0].message
+            if isinstance(message, AIToolCallsMessage):
```

Comment:
imo we should leave tool_call_chunks empty in this case

Reply:
Having every chat model emit tool_call_chunks during streaming provides some uniformity. And this solution will work in a future where Anthropic does stream function calls. But I see how it's wasteful to `.dumps` the parsed args.